### ISSUE TITLE
create a stable release / tag / version

### ISSUE THREAD

**Issue by jackblackCH** [johnpolacek/imagefill.js#15]
At the moment there is not a tagged release which ensures me as developer having a stable version for "life".

Example:
I can't use directly an explicit stable version in bower (bower.json) at the moment. Using code from master is not a solution.

**Comment by camilodelvasto** [comment]
Yes, this is much needed, especially when working with npm/bower

**Comment by tbredin** [comment]
+1
